kotest-assertions/kotest-assertions-core/src/commonMain/kotlin/io/kotest/assertions/nondeterministic/eventually.kt

```diff
@@ -202,7 +207,7 @@ class EventuallyConfigurationBuilder {
 typealias EventuallyListener = suspend (Int, Throwable) -> Unit
 
 object NoopEventuallyListener : EventuallyListener {
-   override suspend fun invoke(iteration: Int, error: Throwable) {}
+   override suspend fun invoke(iteration: Int, error: Throwable, ) {}
```

Comment:
```suggestion
   override suspend fun invoke(iteration: Int, error: Throwable) {}
```

formatting nitpick

Reply:
I personally like trailling comma but since it ruin's commit's visibility I will fix this!

Reply:
I like trailing commas too, but only if the parameters are on separate lines :)